roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1090,7 +1090,7 @@ def set_container_facts_if_unset(facts):
         if 'ovs_image' not in facts['node']:
             facts['node']['ovs_image'] = ovs_image
 
-    if facts['common']['is_containerized']:
+    if facts['common']['is_containerized'] in ['true', 'True']:
```

Comment:
Wouldn't there be other places we'd need this same check for `is_containerized` in openshift_fact.py?  I feel like it would be better to make sure it's set correctly the fact is first set.  That's why I initially thought we should fix the problem in https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_facts/tasks/main.yml#L28 so that everywhere else we could assume that the value was a bool.  The side effect of that is that `is_containerized` would now be set on every host to either true or false instead of the assumption that only a missing value was false.